tests/e2e.sh

```diff
@@ -30,14 +30,18 @@ TESTS_TARGETS=($(bazel query 'tests(//tests/e2e/tests/...)'))
 FAILURE_COUNT=0
 SUMMARY='Tests Summary'
 
-
-
+# query project-id and put on environment
```

Comment:
Let s not make this a default. Let s have the user set it only when testing on GCP.